src/language/dafnyLanguageClient.ts

```diff
@@ -41,16 +41,21 @@ function getMarkGhostStatementsArgument(): string {
 }
 
 export class DafnyLanguageClient extends LanguageClient {
+  private statusOutput: OutputChannel;
+
   // eslint-disable-next-line max-params
-  private constructor(id: string, name: string, serverOptions: ServerOptions, clientOptions: LanguageClientOptions, forceDebug?: boolean) {
+  private constructor(id: string, name: string, serverOptions: ServerOptions,
+      clientOptions: LanguageClientOptions, outputChannel: OutputChannel, forceDebug?: boolean) {
     super(id, name, serverOptions, clientOptions, forceDebug);
+    this.statusOutput = outputChannel;
   }
 
   public getCounterExamples(param: ICounterExampleParams): Promise<ICounterExampleItem[]> {
-    return this.sendRequest<ICounterExampleItem[]>('dafny/counterExample', param);
+    return this.sendLoggedRequest<ICounterExampleParams, ICounterExampleItem[]>(
+      'dafny/counterExample', param);
```

Comment:
I prefer to have the closing parenthesis on a separate line if the parameters are listed on a new line.